website-1.x/blog/2021-03-09-releasing-docusaurus-i18n.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Releasing Docusaurus i18n
+author: Sébastien Lorber
+authorTitle: Docusaurus maintainer
+authorURL: https://sebastienlorber.com
+authorImageURL: https://github.com/slorber.png
+authorTwitter: sebastienlorber
+tags: [release, i18n]
+image: /img/blog/2021-03-09-releasing-docusaurus-i18n/social-card.png
+---
+
+Today, we officially release **[Docusaurus 2 i18n](https://v2.docusaurus.io/docs/i18n/introduction)** ([internationalization](https://en.wikipedia.org/wiki/Internationalization_and_localization)), as part of [2.0.0-alpha.71](https://github.com/facebook/docusaurus/releases/tag/v2.0.0-alpha.71).
+
+Docusaurus 2 has now reached **full feature parity with Docusaurus 1**, and a first **beta** is on its way.
+
+**We went the extra mile**, and the new i18n support is **even better than in Docusaurus 1**.
+
+In this post, we will present you the **translation workflow**, explain some **design decisions** and **showcase early adopter sites** that just went live: [Jest](https://jestjs.io/), [Redwood](https://learn.redwoodjs.com/), and [Datagit](https://datagit.ir/).
+
+<!--truncate-->
+
+## Translate your site
+
+You can translate a Docusaurus site in **3 simple steps**:
+
+1. **Configure**: declare the default locale and alternative locales in `docusaurus.config.js`.
+2. **Translate**: put the translation files at the correct file-system location.
```

Comment:
Nit - file system is either file system or filesystem. file-system is incorrect

Reply:
fixed everywhere